<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(96, 195, 244);</string>
      </property>
      <property name="title">
       <string>SmartCare</string>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>851</width>
         <height>751</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="general">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>341</width>
           <height>371</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(155, 232, 255);</string>
         </property>
         <property name="title">
          <string>service management</string>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>230</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Donquantity</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>90</y>
            <width>91</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>serviceName</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>130</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Budget</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_budget">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>130</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_ajouter">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>300</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(85, 170, 127);</string>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_ID">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>50</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>180</y>
            <width>56</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Dontype</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_name">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>90</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_qnt">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>230</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pb_modifier">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>300</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 170, 255);</string>
          </property>
          <property name="text">
           <string>Modify</string>
          </property>
         </widget>
         <widget class="QComboBox" name="cb_type">
          <property name="geometry">
           <rect>
            <x>112</x>
            <y>180</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>ghta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>shelter</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>iftar</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>money</string>
           </property>
          </item>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>10</y>
           <width>401</width>
           <height>381</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(155, 232, 255);</string>
         </property>
         <property name="title">
          <string>show + PDF + excel + stats</string>
         </property>
         <widget class="QTableView" name="tab_service">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>361</width>
            <height>241</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pb_pdf1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>320</y>
            <width>131</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(85, 170, 127);</string>
          </property>
          <property name="text">
           <string>Generate PDF</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>151</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>List of services :</string>
          </property>
         </widget>
         <widget class="QPushButton" name="stats">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>320</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 170, 255);</string>
          </property>
          <property name="text">
           <string>Stats</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>400</y>
           <width>401</width>
           <height>301</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(155, 232, 255);</string>
         </property>
         <property name="title">
          <string>search + sort + PDF</string>
         </property>
         <widget class="QTableView" name="recherche">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>70</y>
            <width>351</width>
            <height>161</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_recherche">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_recherche">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>30</y>
            <width>101</width>
            <height>22</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>IDservice</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>serviceName</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dontype</string>
           </property>
          </item>
         </widget>
         <widget class="QPushButton" name="pb_recherche">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>30</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 170, 127);</string>
          </property>
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_sort">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>250</y>
            <width>91</width>
            <height>22</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>IDservice</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>serviceName</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Donquantity</string>
           </property>
          </item>
         </widget>
         <widget class="QPushButton" name="pb_sort">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>250</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(85, 170, 127);</string>
          </property>
          <property name="text">
           <string>sort</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pb_pdf2">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>250</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 170, 255);</string>
          </property>
          <property name="text">
           <string>Generate PDF</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>460</y>
           <width>341</width>
           <height>101</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(155, 232, 255);</string>
         </property>
         <property name="title">
          <string>delete service</string>
         </property>
         <widget class="QPushButton" name="pushButton_supprimer">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>40</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 170, 127);</string>
          </property>
          <property name="text">
           <string>delete</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>IDservice</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_idsup">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>40</y>
            <width>101</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="export_excel">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>330</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 170, 127);</string>
         </property>
         <property name="text">
          <string>export excel</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="calendar">
        <attribute name="title">
         <string>Stock Verification</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>70</y>
           <width>591</width>
           <height>431</height>
          </rect>
         </property>
         <property name="title">
          <string>GroupBox</string>
         </property>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>50</y>
            <width>151</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Enter the service's ID:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_idss">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>50</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
         <widget class="QPushButton" name="stockVerification">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>90</y>
            <width>121</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 170, 127);</string>
          </property>
          <property name="text">
           <string>Verify your Stock</string>
          </property>
         </widget>
         <widget class="QTableView" name="Stock">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>150</y>
            <width>441</width>
            <height>241</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(133, 214, 255);</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionlogo"/>
   <addaction name="actionIcon2"/>
   <addaction name="actionicon3"/>
   <addaction name="actionicon4"/>
   <addaction name="actionicon5"/>
   <addaction name="actionIcon1"/>
  </widget>
  <action name="actionIcon1">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/events.png.png</normaloff>:/images/events.png.png</iconset>
   </property>
   <property name="text">
    <string>Icon1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionIcon2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/employees.png.png</normaloff>:/images/employees.png.png</iconset>
   </property>
   <property name="text">
    <string>Icon2</string>
   </property>
  </action>
  <action name="actionicon3">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/destitute.png.png</normaloff>:/images/destitute.png.png</iconset>
   </property>
   <property name="text">
    <string>icon3</string>
   </property>
  </action>
  <action name="actionicon4">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/donation.png.png</normaloff>:/images/donation.png.png</iconset>
   </property>
   <property name="text">
    <string>icon4</string>
   </property>
  </action>
  <action name="actionicon5">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/services.png.png</normaloff>:/images/services.png.png</iconset>
   </property>
   <property name="text">
    <string>icon5</string>
   </property>
  </action>
  <action name="actionlogo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/logo application.png</normaloff>:/images/logo application.png</iconset>
   </property>
   <property name="text">
    <string>logo</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
